classes/helper.php

```diff
@@ -57,8 +57,8 @@ public static function get_columns(string $table, array $searchingcolumns = [],
             return [];
         }
 
-        // Knowns tables to skip.
-        $skiptables = ['grade_grades_history'];
+        // Known tables to skip.
+        $skiptables = ['grade_grades_history', 'notifications', 'question_attempt_step_data'];
```

Comment:
Lets turn this into a new command line option --exclude-tables so we don't hard code it and can more easily add more on the fly

I think grade_grades_history should always be skippedso leave it hard coded. It should be hard coded in core in db_should_replace but we can worry about that later